packages/vats/src/vat-bank.js

```diff
@@ -882,25 +881,27 @@ export function buildRootObject(_vatPowers, _args, baggage) {
         'denomToAddressUpdater',
       );
 
-      /** @param {ERef<import('./types.js').ScopedBridgeManager<'bank'>>} [bankBridgeMgr] */
-      async function getBankChannel(bankBridgeMgr) {
+      async function getBankChannel() {
```

Comment:
Does this change mean `vat-bank` needs to be upgraded with the next release?

Reply:
It's a refactor that shouldn't affect the behavior. @michaelfig WDYT, should I revert?

Reply:
I haven't looked carefully, but if the only call to `getBankChannel` is on line 903-904, I'm okay with the refactor.